UKHomeOffice/nmsw-ui
# NMSW-1296 Extend skip link css to fill entire page

# Ticket

NMSW-1296

----

## AC

Currently the skip link is a button and it does not extend the length of the page. This PR extends the focus state css so that the skip banner fills the length of the screen.

----

## To test

- Run app
- Tab in from the URL
- > You should see the skip link extending to the full width of the screen
- > It should skip to the main content on click/enter


----

## Notes
